.github/CODEOWNERS

```diff
@@ -308,1068 +308,986 @@ app/models/veteran_onboarding.rb @department-of-veterans-affairs/backend-review-
+docs/setup/hybrid.md @department-of-veterans-affairs/backend-review-group
+docs/setup/images @department-of-veterans-affairs/backend-review-group
+docs/setup/local_network_access.md @department-of-veterans-affairs/backend-review-group
+docs/setup/mailer.md @department-of-veterans-affairs/backend-review-group
+docs/setup/mhv.md @department-of-veterans-affairs/vfs-health-modernization-initiative @department-of-veterans-affairs/backend-review-group
 docs/setup/mpi.md @department-of-veterans-affairs/octo-identity
-docs/setup/native.md @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers
-docs/setup/rswag_setup.md @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers
-docs/setup/running_binstubs.md @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers
-docs/setup/running_docker.md @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers
-docs/setup/running_natively.md @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers
-docs/setup/setup_with_binstubs.md  @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers
-docs/setup/virtual_machine_access.md @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/va-api-engineers
+docs/setup/native.md @department-of-veterans-affairs/backend-review-group
+docs/setup/rswag_setup.md @department-of-veterans-affairs/backend-review-group
+docs/setup/running_binstubs.md @department-of-veterans-affairs/backend-review-group
+docs/setup/running_docker.md @department-of-veterans-affairs/backend-review-group
+docs/setup/running_natively.md @department-of-veterans-affairs/backend-review-group
+docs/setup/setup_with_binstubs.md @department-of-veterans-affairs/backend-review-group
+docs/setup/virtual_machine_access.md @department-of-veterans-affairs/backend-review-group
 Gemfile @department-of-veterans-affairs/backend-review-group
 Gemfile.lock @department-of-veterans-affairs/backend-review-group
 helmCharts/ @department-of-veterans-affairs/backend-review-group
 import-va-certs.sh @department-of-veterans-affairs/backend-review-group
 Jenkinsfile @department-of-veterans-affairs/backend-review-group
+lib/accredited_representation/ @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/benefits-accredited-rep-facing-engineers @department-of-veterans-affairs/accredited-representatives-admin
 lib/accredited_representation/constants.rb @department-of-veterans-affairs/accredited-representation-management @department-of-veterans-affairs/backend-review-group @department-of-veterans-affairs/benefits-accredited-rep-facing-engineers @department-of-veterans-affairs/accredited-representatives-admin
```

Comment:
What do you think of deleting this line and adding `accredited-representation-management` to the line above?